caffe2/proto/caffe2.proto

```diff
@@ -11,27 +11,73 @@ package caffe2;
 //     is mainly for backward compability purposes but we may consider using
 //     those in the future.
 
+// ExternalDataProto stores the pointer to the content of TensorProto
+// the content are stored in the raw format as little endian
+message ExternalDataProto {
+  // type of the external storage type, can be the following:
+  enum SourceType {
+    // use external file to store the data, and id is the POSIX relative path
+    // to the file
+    FILE = 0;
+    // the wrapper defined in torch/csrc/jit/serialization.h is used,
+    // and id is the tag to help the runtime identify the data
+    // this type of storage is set as DEFAULT and recommended for external
+    // data storage
+    TORCH_WRAPPER = 1;
```

Comment:
maybe call it CONTAINER or INLINE_CONTAINER. TORCH_WRAPPER is kind of a weird name

Reply:
Yeah, INLINE_CONTAINER sounds much better.